dynamodb/package-lock.json

```diff
@@ -17,413 +17,4774 @@
         "aws-sdk": "^2.1610.0"
       }
     },
-    "node_modules/@aws-crypto/crc32": {
+    "node_modules/@aws-crypto/sha256-browser": {
```

Comment:
I ran an `npm audit fix` which seemed to have made most of these tweaks. Let me know if I should revert this!